sandbox/registration-processor-mz.properties

```diff
@@ -128,6 +128,8 @@ RIDGENERATION=http://kernel-ridgenerator-service/v1/ridgenerator/generate/rid
 # We specify Ingress location for dmz which is any worker node and a ingress port
 SYNCSERVICE=${dmz.ingress.base.url}/registrationprocessor/v1/registrationstatus/sync
 PACKETRECEIVER=${dmz.ingress.base.url}/registrationprocessor/v1/packetreceiver/registrationpackets
+# this property is used in lostrid api to get postal code 
+mosip.registration.processor.postalcode.req.url=${dmz.ingress.base.url}/v1/masterdata/registrationcenters
```

Comment:
Master data is not running on dmz, please update as below

mosip.registration.processor.postalcode.req.url=${MASTER}/registrationcenters

Reply:
taken care

Reply:
ok resolved